src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,101 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  max-width: getContainerWidth($card-width, $card-gap, 40);
+  padding: 50px 40px;
+
+  &--centered {
+    margin: 0 auto;
+  }
+  &__card {
+    margin-right: $card-gap;
+    margin-bottom: 46px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  background: #fff;
```

Comment:
```suggestion

  background: #fff;
```

Reply:
Thank you for your review. Will do as suggested. @ysrckr